tempesta_fw/addr.c

```diff
@@ -44,48 +45,60 @@ validate_addr(const TfwAddr *addr)
 }
 
 static int
-tfw_inet_pton_ipv4(char **p, struct sockaddr_in *addr)
+tfw_inet_pton_ipv4(const char *pos, struct sockaddr_in *addr)
```

Comment:
As far as I can remember, the old function was not able things like ":80", so I wrote a new one. That turned out to be faster than enhancing the old implementation.